src/components/common/stateIcon.scss

```diff
@@ -1,6 +1,4 @@
 .resource-state-text {
-  margin-right: 1rem;
```

Comment:
But anyway - leaving it up to you - since it's not regressing I am fine with that as well.

Reply:
Both of these are deliberate. Removing  `margin-right: 1rem;` otherwise items will be spaced far too apart from each other, which is opposite of the design mockup: https://github.com/cockpit-project/cockpit-machines/pull/998#pullrequestreview-1370865494
![Screenshot from 2023-04-05 20-35-07](https://user-images.githubusercontent.com/42733240/230173867-df288290-e442-4ca7-8d02-44635626cfbe.png)

Reply:
Ahhh, now I noticed that without that margin-right, it will break state in vmDetailsPage.... Aaahh, css, I'll look at that more
![Screenshot from 2023-04-05 20-44-55](https://user-images.githubusercontent.com/42733240/230175729-8cc01af3-b289-43a0-b7de-88dc4cd59451.png)

Reply:
Ok, I changed the css a bit again. I cannot revert all the changes to the file, since we want to increase the spaces to xs as discussed in [mockup](https://github.com/cockpit-project/cockpit-machines/pull/998#pullrequestreview-1370865494)